docs/sphinx/ome-files-env.txt

```diff
@@ -76,10 +76,10 @@ basic paths, above.
 
 :envvar:`OME_FILES_DATADIR`
   OME-Files data files
-:envvar:`OME_FILES_ICONDIR`
-  OME-Files icons
 :envvar:`OME_FILES_LIBEXECDIR`
   OME-Files program executables
+:envvar:`OME_QTWIDGETS_ICONDIR`
```

Comment:
Yes.  This is the documentation for *all* of the components.  We could potentially split this out as well.